hardware/detect.py

```diff
@@ -726,6 +731,7 @@ def _get_governor(lcpu):
                                      ('family', 'CPU family', int),
                                      ('model', 'Model', _maybe_int),
                                      ('stepping', 'Stepping', _maybe_int),
+                                     ('architecture', 'Architecture', None),
```

Comment:
I don't see how this is correlated to the patch

Reply:
The patch adds two fields to the output: CPU architecture field and SMT status to the collected data. They were two independent patches that I squashed as suggested by @elfosardo. I updated the title of the merge request to make it less confusing.

Reply:
Yeah sorry I missed that.